vpr/src/base/vpr_constraints.xsd

```diff
@@ -33,6 +33,8 @@
     <xs:attribute name="y_low" type="xs:int" use="required"/>
     <xs:attribute name="x_high" type="xs:int" use="required"/>
     <xs:attribute name="y_high" type="xs:int" use="required"/>
+    <xs:attribute name="layer_low" type="xs:int"/>
```

Comment:
Write a validator somewhere after the line or file is read in that layer_low <= layer_high, and both are >= 0. If you know the grid size at that point, also check both are in the grid.

Check that the xlow, etc. values are also validated in the same way.